dg/commands/ibmcloud/logout.py

```diff
@@ -19,12 +19,13 @@
 import dg.config
 
 from dg.commands.ibmcloud.common import is_logged_in
+from dg.config import data_gate_configuration_manager
```

Comment:
Same here : 
```suggestion
from dg.config.data_gate_configuration_manager import get_ibmcloud_cli_path
```

Reply:
I removed `import dg.config` and replaced `dg.config.data_gate_configuration_manager` with `data_gate_configuration_manager`.